README.md

```diff
@@ -311,9 +272,33 @@ Attributes Kit always takes the same parameters.
 
 **Congratulations!** You are all set to start developing now! :rocket:
 
-## Playground
+---
+
+# Compatibility
 
-## Testing
+| Package                 | Version |
+|:------------------------|:--------|
+| React | `0.14+` |
+| React DOM | `0.14+` |
+| Node | `0.10.40+` |
```

Comment:
@XVincentX 

Node is required for development and server side rendering, am I right?